src/java/htsjdk/variant/variantcontext/Allele.java

```diff
@@ -307,8 +323,8 @@ public static boolean acceptableAlleleBases(byte[] bases, boolean allowNsAsAccep
             switch (base) {
                 case 'A': case 'C': case 'G': case 'T':  case 'a': case 'c': case 'g': case 't': 
                     break;
-                case 'N' : case 'n' :
-                    if (allowNsAsAcceptable)
+                case 'N' : case 'n' : case '*':
+                    if (isReferenceAllele)
                         break;
```

Comment:
This needs to be changed.
1) It turns out that the spec was changed and 'N's are allowed in both the REF and ALT fields, so they should be treated just like other bases above.
2) '*'s are acceptable only in the ALT field (so NOT if isReferenceAllele is true).  But they are also only allowed as the ONLY character (so this check needs to occur outside of the for loop).  E.g.
if ( wouldBeStarAllele(bases) )
             return true;
and wouldBeStarAllele() would check that it's a single base == to '*'.